packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -412,14 +409,12 @@ export class TeleReporterReceiver {
   private _absolutePath(relativePath: string): string;
   private _absolutePath(relativePath?: string): string | undefined;
   private _absolutePath(relativePath?: string): string | undefined {
-    if (!relativePath)
-      return relativePath;
```

Comment:
I believe we need to change this check to `if (relativePath === undefined)` instead.